pkg/storage/system_intake.go

```diff
@@ -518,7 +462,7 @@ func (s *Store) FetchSystemIntakesByStateForAdmins(ctx context.Context, state mo
 		SELECT *
 		FROM system_intakes
 		WHERE state=$1
-		AND (status != 'WITHDRAWN' AND archived_at IS NULL)
```

Comment:
double check

Reply:
removed a hard-coded conditional in SQL - is this ok?

Reply:
I think for the same reason as my last (silly) comment, yes, this should be alright!